gridlab-d/gridlab-d
# Issue/1272

#### What's this Pull Request do?
Addresses https://github.com/gridlab-d/gridlab-d/issues/1272: `measured_avg_voltage_<phase>_in_interval` meter properties were not working and this PR fixes that.

#### Where should the reviewer start?
View diff to meter.cpp.

#### How should this be tested?
Run the new test in `powerflow/autotest`: `test_ami_average_voltage_features_NR.glm`. All other existing tests should now be working.

Anyone interested in the script used to generate player files and expected results .csvs for the test should see the included Python script, `gen_data_for_test_ami_average_voltage_features.py`.

#### Any background context you want to provide?
Recommend squashing commits into a single commit during the merging process.

#### What are the relevant issues?
https://github.com/gridlab-d/gridlab-d/issues/1272

#### Screenshots (if appropriate)
N/A

#### Questions:
- [x] Does this add new dependencies? **No**
- [x] Is there appropriate logging included? **N/A**
